apis/mattermost/v1alpha1/clusterinstallation_types.go

```diff
@@ -239,6 +239,9 @@ type Database struct {
 	// Defines the secret to be used when performing a database restore.
 	// +optional
 	BackupRestoreSecretName string `json:"backupRestoreSecretName,omitempty"`
+	// Defines the cluster version for the database to use
+	// +optional
+	Version string `json:"version,omitempty"`
```

Comment:
@fmartingr, It looks like this change has nothing to do with this migration. Do you want to move it (with the relevant code) to a separate pr?